### PR TITLE
Update README and setup script for Mediatek Platform

### PR DESCRIPTION
**Pull request by jamescfkung**
Signed-off-by: James Kung <james.kung@linaro.org>

### CODE REVIEW

**Comment by jbech-linaro** [comment]
@jamescfkung I will try to follow your description here. When I've done so, I'll add my reviewed-by tag.

File: README.md [hunk 1]
```diff
@@ -565,6 +566,67 @@ $ /system/bin/tee-helloworld
...
+
+#### 4.7.3 Update MT8173 EVB firmware images
+Run `flash_image.sh` to update MT8173 EVB firmware images
+```
+$ ./flash_image.sh
+```
+
+#### 4.7.4 Firmware recovery
+
+  1. Donwload pre-built images and recovery tools
```
Comment (jbech-linaro) [inline README.md]: Spelling: "Download"

File: README.md [hunk 2]
```diff
@@ -565,6 +566,67 @@ $ /system/bin/tee-helloworld
...
+  ```
+  $ git clone https://github.com/m943040028/evb-utils.git
+  $ cd evb-utils
+  $ ./get-fbtool.sh
+  ```
+
+  2. Force EVB to enter fastboot mode (root privilege required)
```
Comment (jbech-linaro) [inline README.md]: Add "s": privileges

**Comment by jamescfkung** [comment]
Typo fixed

**Comment by jbech-linaro** [comment]
`Reviewed-by: Joakim Bech <joakim.bech@linaro.org>`
`Tested-by: Joakim Bech <joakim.bech@linaro.org> (MT8173)`

**Comment by pascal-brand-st-dev** [comment]
Reviewed-by: Pascal Brand <pascal.brand@linaro.org>

**Comment by pascal-brand-st-dev** [comment]
Manually merged

**Comment by pascal-brand-st** [comment]
Manually merged

**Pull request closed without merging**